pennylane/qnode.py

```diff
@@ -519,10 +518,7 @@ def transform_program(self):
         return self._transform_program
 
     def add_transform(self, transform_container):
-        """Add a transform container to the transform program.
-
-        .. warning:: This is an experimental feature.
-        """
+        """Add a transform container to the transform program."""

```

Comment:
Non-blocking point but this could do with a more elaborated docstring.

Reply:
I added:

```
        .. warning:: This is developer facing feature and is called when transforms are applied on a QNode.
```